README.md

```diff
@@ -22,7 +23,7 @@ JSON input and generate the REGO policy.
 $ npm install -g tree-sitter-eventrule
 $ rule2rego --help
 Compiles AWS Event Rule pattern JSON to OPA REGO policy.
-Usage: rule2rego <rule>.json
+Usage: rule2rego [<rule>.json] [folder]
```

Comment:
The interface is now kinda awkward. what if the tool accepted multiple jsons on its input and treated them as arrays of rules instead? So you can either pass one or multiple.

like `rule2rego one.json two.json three.json` etc.

Then the expansion of json can be done through the shell.

like `bash -c "rule2rego ./**/*.json"`